pkg/webhook/policy.go

```diff
@@ -338,12 +338,20 @@ func (h *validationHandler) getValidationMessages(res []*rtypes.Result, req *adm
 func (h *validationHandler) validateGatekeeperResources(ctx context.Context, req *admission.Request) (bool, error) {
 	gvk := req.AdmissionRequest.Kind
 
+	// for resources that don't have a name validation
```

Comment:
Why not just run this validation for every resource before the switch{} statement?